Updated components for mentioning layer deletion in Adobe XD (#537)
IgniteUI/indigo-design-docfx: en/components/linear-gauge.md

## Title and Subtitle

> The Linear Gauge has a Title and Subtitle whose text can be changed. In Sketch, they can also be hidden if they are not needed by setting them to ~No Symbol, while in Adobe XD you have to delete the unused layer instead.

Reviewer: ```suggestion
The Linear Gauge has a Title and Subtitle whose text can be changed. In Sketch, they can also be hidden if they are not needed by setting them to ~No Symbol, while in Adobe XD you should delete their respective layers.
```

<img class="responsive-img" src="../images/linear_gauge_title.png" srcset="../images/linear_gauge_title@2x.png 2x" />

## Legend

> The Linear Gauge comes with a Legend that can be adjusted accordingly to match the ranges shown on it and the current value. In Sketch, Legend items can be removed by setting the corresponding Series to ~No Symbol or added by finding it in the Overrides/Charts/Legend/Serie. In Adobe XD, you have to delete the unused layers instead, and the layout will adjust through the use of stacks. Also, you can change the icon type between Square, Line, or Circle.

Reviewer: ```suggestion
The Linear Gauge comes with a Legend that can be adjusted accordingly to match the ranges shown on it and the current value. In Sketch, Legend items can be removed by setting the corresponding Series to ~No Symbol or added by finding it in the Overrides/Charts/Legend/Serie. In Adobe XD, you have to delete the unused Series layers, and the layout will adjust automatically. Additionally, you can change the icon type between Square, Line, or Circle.
```

<img class="responsive-img" src="../images/linear_gauge_legend.png" srcset="../images/linear_gauge_legend@2x.png 2x" />

## Tooltip

> The Linear Gauge comes with a tooltip that may be hidden by setting the override to ~No Symbol in Sketch, while in Adobe XD you have to delete the unused layer instead.

Reviewer: ```suggestion
The Linear Gauge comes with a tooltip that may be hidden by setting the override to ~No Symbol in Sketch or deleting the layer for it in Adobe XD.
```
